addons/mail/static/src/js/mail.js

```diff
@@ -1190,20 +1232,20 @@ var ComposeMessage = Attachment.extend ({
      * post a message and fetch the message
      */
     do_send_message_post: function (partner_ids, log) {
-        var self = this;
+        var self = this, selected_partners_ids = _.keys(this.selected_partners);
 
         var values = {
-            'body': this.$('textarea').val(),
+            'body': this.mail_body,
             'subject': false,
             'parent_id': this.parent_id || this.id,
-            'partner_ids': partner_ids,
+            'partner_ids': _.uniq(partner_ids.concat(selected_partners_ids)),
             'attachment_ids': _.map(this.attachment_ids, function (file) {return file.id;}),
             'context': _.extend(this.parent_thread.context, {
                 'mail_post_autofollow': true,
                 'mail_post_autofollow_partner_ids': partner_ids,
             }),
             'message_type': 'comment',
-            'content_subtype': 'plaintext',
+            'content_subtype': 'html',
```

Comment:
I think this is the type by default, serverside on message_post(). If so, remvoe this line.